WalletWasabi.Tests/UnitTests/Wallet/WalletBuilder.cs

```diff
@@ -57,16 +60,17 @@ public async Task<WalletWasabi.Wallets.Wallet> CreateRealWalletBasedOnTestWallet
 		keyManager.GetKeys(_ => true); // Make sure keys are asserted.
 
 		var serviceConfiguration = new ServiceConfiguration(new UriEndPoint(new Uri("http://www.nomatter.dontcare")), Money.Coins(WalletWasabi.Helpers.Constants.DefaultDustThreshold));
-		WasabiSynchronizer synchronizer = new(requestInterval: TimeSpan.FromSeconds(3), 1000, BitcoinStore, HttpClientFactory);
-		HybridFeeProvider feeProvider = new(synchronizer, null);
+
+		HybridFeeProvider feeProvider = new(Synchronizer, null);
 		SmartBlockProvider blockProvider = new(BitcoinStore.BlockRepository, rpcBlockProvider: null, null, null, Cache);
 
-		return WalletWasabi.Wallets.Wallet.CreateAndRegisterServices(Network.RegTest, BitcoinStore, keyManager, synchronizer, DataDir, serviceConfiguration, feeProvider, blockProvider);
+		return WalletWasabi.Wallets.Wallet.CreateAndRegisterServices(Network.RegTest, BitcoinStore, keyManager, Synchronizer, DataDir, serviceConfiguration, feeProvider, blockProvider);
 	}
 
 	public async ValueTask DisposeAsync()
 	{
 		await IndexStore.DisposeAsync().ConfigureAwait(false);
+		await Synchronizer.StopAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
```suggestion
		await Synchronizer.StopAsync(CancellationToken.None).ConfigureAwait(false);
		Synchronizer.Dispose();
```

Reply:
StopAsync does the same that Dispose do.

Reply:
True.